robocop/checkers/naming.py

```diff
@@ -75,8 +75,9 @@ class KeywordNamingChecker(VisitorChecker):
     rules = {
         "0302": (
             "not-capitalized-keyword-name",
-            "Keyword name should be capitalized",
-            RuleSeverity.WARNING
+            "Keyword name should use title case",
+            RuleSeverity.WARNING,
+            ('check_only_first_word', 'check_only_first_word', bool)
```

Comment:
How about naming this parameter as `convention` (or whatever fits better) and allowing values like:
- `first_word_capitalized` - only first word needs to be capitalized (like in `This is my keyword`)
- `each_word_capitalized` - each word in keyword name needs to be capitalized (like in `This Is My Keyword`)
- `none` - no rules for case in keyword naming, so even names like `this_is_my_keyword` are allowed

Reply:
I agree. Does none means all lower case? So it will report on 'First word', and 'All Words' but not on "all lowercase'. Because if it means 'don't check case' then it would be the same as disabling this rule.

Reply:
I actually meant that 'none' means that nothing is required from the keyword name but you're right that it is the same as disabling this rule. Should we have also some other values then `first_word_capitalized` and `each_word_capitalized`?

Reply:
`first_word_capitalized`, `each_word_capitalized` sounds fine, although it could be also convention: `capitalize`, `title_case`, `lower_case`

Reply:
Can you elaborate on those modes? Do you consider them as something different or they are just different names for the ones I explained?

Reply:
Just different names

Reply:
I'm still not sure... Mine are more descriptive, but yours are shorter 😁

Reply:
I will go with yours, it's better to be more descriptive